app/models/user.rb

```diff
@@ -69,6 +71,10 @@ def password_reset_expired?
     reset_sent_at < 2.hours.ago
   end
 
+  def feed
+    Micropost.micropost_ordered
```

Comment:
viết như thế này thì cái feed nó trả về toàn bộ micropost của tất cả các user à ?